WatchFaceAlphaKotlin/app/src/main/java/com/example/android/wearable/alpha/AnalogWatchCanvasRenderer.kt

```diff
@@ -203,6 +210,7 @@ class AnalogWatchCanvasRenderer(
     }
 
     override fun onDestroy() {
+        Log.d(TAG, "onDestroy()")
```

Comment:
Nit: did you want to remove this one too?

Reply:
Removed